### PR TITLE
Create section dividers and refactor code

### PR DESCRIPTION
**Pull request by matheushss1**
### Global CSS:

Added dividers CSS. (f65030f)
Adjusted color hex values to lowercase, added missing semicolon, and ensured proper formatting. (57015be)
Removed the dark mode media query to prevent display issues. (0b9cfd0),

### Components:

Moved several components (e.g., Footer, Hero, Testimonials, Schedule) into the sections directory. Added a backgroundColor prop and adjusted padding, margins, and height for better layout consistency. (b6321ee, f3ebee7, 948f43a, 2722002, 6eebc42, 65efed0, 71c4f99)
Adjusted naming conventions and spacing across components to improve readability and maintainability. (292f061)

### Page-Level Changes:

Referenced section components instead of writing sections directly, enabling cleaner and more modular code. Added dividers between sections. (b99ee22)
Improved import formatting (e.g., switched to double quotes) and added spacing for readability. (0eb94de, 47eed82)

### Layout Adjustments:

Improved spacing and formatting in layout.js for better readability. (6ee4623)

## In-progress:
- [x] Create component for dividers.
- [x] Adjust hero background cover image.
- [x] Responsiveness (mobile)

### CODE REVIEW

File: app/page.js [hunk 1]
```diff
@@ -16,145 +15,35 @@ export default function Home() {
...
 
-        <section id="how-we-work" className="bg-white">
-          <div className="container grid grid-cols-2 gap-8 py-8">
-            
-            <EngagementAccordion />
+        <div className={`relative bg-platinum`}>
+          <div
+            className={`relative w-full h-10 sm:h-20 bg-platinum start-curve-divider`}
+          ></div>
+        </div>
```
Comment (am17torres) [inline app/page.js:23]: Should this be using the Divider Component?

**Review by am17torres — COMMENTED** [review]
Looks good. There are a few small changes - 

image

image

Is there some fixed height on the schedule section? I noticed the bottom gets clipped when the vw is small. In the second screenshot the last accordion item gets hidden behind the divider

File: app/page.js (hunk 1 above)
Comment (matheushss1) [reply app/page.js:23]: Yes, it should. But there's some weird behavior that is making the divider don't appear when I use the Divider component there. My best guess is that there's something to do with the hero component. Should this fix be in the scope of this PR?

**Comment by matheushss1** [comment]
> Is there some fixed height on the schedule section? I noticed the bottom gets clipped when the vw is small. In the second screenshot the last accordion item gets hidden behind the divider

Can you take a look now to see if it's better?

**Review by am17torres — APPROVED** [review]

**Pull request merged**